### PR TITLE
Fix using NoisyNet with .fit() or .train_on_batch()

### PR DESCRIPTION
**Pull request by markub3327**
The standard Keras layer `call()` method contains only **inputs, training and mask arguments**. Using reset_noise() and remove_noise() outside the call() is an effective way when we decide to save the model in h5 format (We don't just have to use a custom model).
 
For loading model:             
`model = tf.keras.models.load_model(fileName, compile=False, custom_objects={'NoisyDense': NoisyDense})`

When eps_bias and eps_kernel weren't defined by `add_weight()` method the learning process doesn't work for me and agent not converge (the agent not achieve a game goal).

Thanks.

### CODE REVIEW

**Comment by markub3327** [comment]
@bhack Yes, I can maintain this layer. Thanks.

**Comment by bhack** [comment]
Sorry, please commit here

**Comment by markub3327** [comment]
@bhack  Sorry, now it's done ...
